_includes/footer.html

```diff
@@ -9,6 +9,9 @@ <h2 class="text-xl text-yellow-600">Get Started</h2>
                 <a class="footer-menu" href="{{ "/docs/get-involved" | relative_url }}">
                     <i class="fas fa-users pr-2"></i>Get Involved
                 </a>
+                <a class="footer-menu" href="https://linkedin.com/company/{{ site.linkedin }}">
+                    <i class="fab fa-twitter pr-2"></i>LinkedIn
```

Comment:
`fa-linkedin` Font awesome icon for LinkedIn

Reply:
https://fontawesome.com/icons/linkedin

Reply:
Done

Reply:
Looks like Twitter and Linkedin icons interchanged